autoload/ddc/map.vim

```diff
@@ -78,6 +78,19 @@ function ddc#map#insert_item(number, cancel_key) abort
   " Call CompleteDone later.
   autocmd ddc TextChangedI * ++once doautocmd <nomodeline> CompleteDone
 
+  " Avoid side effects of ddc-cmdline
+  if mode() ==# 'c'
```

Comment:
You must check `setcmdline()` exists.

Reply:
denops.vim requires Vim 9.0.1499 / Neovim 0.8.0.
`setcmdline()` is added in Vim 9.0.0285 / Neovim 0.8.0

Reply:
OK.  I will update the requirements.